cmd/root.go

```diff
@@ -43,6 +49,9 @@ or servererror`,
 	}
 )
 
+func init() {
+	codeCmd.PersistentFlags().Bool("print", false, "Prints respective rfc")
```

Comment:
wdyt if instead of using `--print` we would use `--print-rfc`?

Reply:
yeah, it will become clearer
anyway, we can change it anytime

Reply:
Should i change it to "print-rfc" or ?
What's the next step ?